sailthru/__init__.py

```diff
@@ -1,6 +1,8 @@
-from sailthru_client import SailthruClient
-from sailthru_error import SailthruClientError
-from sailthru_response import SailthruResponse, SailthruResponseError
+from __future__ import absolute_import, division, print_function, with_statement, unicode_literals
```

Comment:
Done. Also added support/tests for python2.6 by re-adding the use of `simplejson` when `json` is not available.